# Service used to rename a given map.

string old_name
string new_name
---
